requirements-travis.txt

```diff
@@ -1,5 +1,6 @@
 # All pip installable requirements pinned for Travis CI
 pycodestyle==2.4.0
+pylint==1.9.2
```

Comment:
well it'd be nice to keep testing the latest greatest as well. What would you say about including `python 3.7-dev` with the latest requirements? I mean our `requirements.txt` uses `>=` so we should check both...

Reply:
It also looks like this version does not produces the warnings in case it fails to load the module, because you can see the CI passed even though it failed to execute some of the tests.